services/auth/authUser.js

```diff
@@ -0,0 +1,48 @@
+const { jwtSign } = require('../../handlers');
+
+// our user "table"
+const users = [
+  {
+    useremail: 'someone@nowhere.com',
+    password: 'qwerty', // obviously this should be hashed
+    role: 'admin',
+  },
+  {
+    useremail: 'phantom@blackhole.com',
+    password: 'foobar',
+    role: 'editor',
+  },
+  {
+    useremail: 'joedirt@mindspring.com',
+    password: 'mullethead',
+    role: 'reader',
+  },
+];
+
+const findUser = ({ useremail: usr, password: pwd }) => {
+  const foundUser = users.filter(user => user.useremail === usr && user.password === pwd);
+  return foundUser.length === 0 ? null : foundUser[0];
+};
+
+const authUser = (payload) => {
+  const user = findUser(payload);
+  const responseObj = {};
+  if (!user) {
```

Comment:
This is personal preference, but I think it's easier to read if you construct the object on the return call, like so:

```
if (!user) {
  return { success: false, ... };
}
return { success: true };
```